pyproject.toml

```diff
@@ -198,7 +195,7 @@ show_error_codes = true
 [tool.black]
 line-length = 100
 preview = true
-required_version = "24.4.1"
+required-version = "24.4.1"
```

Comment:
Apparently either one works? This seems more consistent with the online black documentation, though.